sklearn/cluster/_spectral.py

```diff
@@ -528,8 +528,8 @@ def fit_predict(self, X, y=None):
 
         Parameters
         ----------
-        X : array-like or sparse matrix, shape (n_samples, n_features), or \
-            array-like, shape (n_samples, n_samples)
+        X : {array-like, sparse matrix} of shape (n_samples, n_features), or \
+            array-like of shape (n_samples, n_samples)
```

Comment:
Same here (over-indented line for visual indent).

Reply:
ok